src/sass/base/components/schools-look-up/_schools-look-up.scss

```diff
@@ -36,19 +36,19 @@
 }
 .schoolsLookUpForm {
   max-width: 100%;
-  @extend input[type="text"];
+  @extend input, [type=text] !optional;
```

Comment:
Hm, not happy about this as you're changing the selector; from just looking at changes you've made it should maybe just be:

`@extend input[type=text];`

Reply:
Ah, I see that the error specifically recommends this; will check out what's changed

Reply:
Before:
![Screenshot 2023-08-15 at 09 36 13](https://github.com/comicrelief/storybook/assets/4737220/8266fe3c-8343-4c96-9649-24960ddb8de1)


After:
![Screenshot 2023-08-15 at 09 35 46](https://github.com/comicrelief/storybook/assets/4737220/de0f976b-9388-488a-866a-b004df58a801)


Always gotta check your changes, my dude; just because something has stopped erroring, doesn't mean it's sorted.

Reply:
Right okay, it seems like the comma approach thing doesn't actually give us complete parity (the computed rules look different and there's some tiny line height change or something), but it's nothing I'm worried about.

Soooo, does  @extend input, [type="text"]; build okay for you? I can't find any reference to that `!optional` flag online?